tests/integration/test_writes/test_writes.py

```diff
@@ -2089,6 +2091,7 @@ def test_read_write_decimals(session_catalog: Catalog) -> None:
     assert tbl.scan().to_arrow() == arrow_table
 
 
+@pytest.mark.skip("UUID BucketTransform is not supported in Spark Iceberg 1.9.2 yet")
```

Comment:
is this test skip related to the trino work?
I am curious why this wasn't skipped before

Reply:
The change lgtm! Thanks, I'll see if I have some time to go over the changes in detail to properly review but we will still need a committer review :)